app/__init__.py

```diff
@@ -9,35 +9,34 @@
 from .models import dbconnect, createTables
 from instance.config import app_configurations
 from .views.product import Products, SingleProduct
-# from views.user import Users, Signin
+from .views.user import Users, Signin
 from .views.sale import Sales
 
 
-v1 = Blueprint("v1", __name__, url_prefix="/api/v1")
-api = Api(v1)
+# v1 = Blueprint("v1", __name__, url_prefix="/api/v1")
```

Comment:
Avoid having commented out blocks of code in your code if you are not using them